martin/src/sprites/mod.rs

```diff
@@ -186,14 +187,19 @@ async fn parse_sprite(
     let tree = Tree::from_data(&buffer, &Options::default())
         .map_err(|e| SpriteParsingError(e, path.clone()))?;
 
-    let sprite = Sprite::new(tree, pixel_ratio).ok_or_else(|| SpriteInstError(path.clone()))?;
+    let sprite = if as_sdf {
+        Sprite::new_sdf(tree, pixel_ratio).ok_or_else(|| SpriteInstError(path.clone()))?
```

Comment:
unsure if I should add another error variant..
Looking at spreet, the failiour cases are similar-ish.

=> accuracy vs semver-breakage
=> your call @nyurik